package.json

```diff
@@ -4,13 +4,15 @@
   "version": "0.0.0",
   "type": "module",
   "engines": {
-    "node": "20"
+    "node": "22"
   },
   "engineStrict": true,
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
     "lint": "eslint --report-unused-disable-directives --max-warnings 0",
+    "test": "vitest",
```

Comment:
I think we should use `"test": "vitest run",`. `vitest` does not quit the process.

Reply:
Will change